src/engine/QueryPlanner.cpp

```diff
@@ -27,6 +27,7 @@
 #include <engine/TransitivePath.h>
 #include <engine/Union.h>
 #include <engine/Values.h>
+#include <engine/WordIndexScan.h>
```

Comment:
Can you please refactor those to follow the format `#include "engine/....h"` (quotes instead of angles for local includes from inside QLever.